charts/team-ns/templates/_ingress.tpl

```diff
@@ -125,15 +126,11 @@ spec:
         - {{ $domain }}
           {{- if hasKey $secrets $domain }}
             {{- if ne (index $secrets $domain) "" }}
-{{/*If a team provides its own certificate in the team namespace then Otomi cornjob makes a copy of it*/}} 
+{{/*If a team provides its own certificate in the team namespace then Otomi cronjob makes a copy of it*/}} 
       secretName: copy-team-{{ $v.teamId }}-{{ index $secrets $domain }}
             {{- end }}
           {{- else }}
-            {{- if eq $cm.issuer "byo-wildcard-cert" }}
-      secretName: otomi-byo-wildcard-cert
-            {{- else }}
-      secretName: otomi-cert-manager-wildcard-cert
-            {{- end}}
+      secretName: {{ $derived.tlsSecretName }}
```

Comment:
A bit nit-picky but my preference would be to keep it consistent with the rest:
```suggestion
      secretName: {{ $v._derived.tlsSecretName }}
```
And then remove the line above.

Reply:
The linter flagged this as an error. This will have to be changed again, because it does not seem to work with team values.